src/backend/cdb/dispatcher/cdbgang_async.c

```diff
@@ -52,6 +53,7 @@ cdbgang_createGang_async(List *segments, SegmentType segmentType)
 	Gang	*newGangDefinition;
 	int		create_gang_retry_counter = 0;
 	int		in_recovery_mode_count = 0;
+	int		other_failures;
```

Comment:
```suggestion
	int		other_failures = 0;
```
Do we also want to initialize it to 0?

Reply:
Actually, `other_failures = 0;` is initialized a few lines below under the `create_gang_retry` goto label.
So, we don't need it here.

Reply:
I saw that, but we also have the `successful_connections` initialized in the `create_gang_retry` block, and also initialized here.

I like the consistency of the coding style, as well as a clear initialization to the local variables in the function.

Reply:
Sounds good, made the change!